### PR TITLE
Card: Added demos for log and event views

### LINKED ISSUE #6287: Card: Log view demo
Add log view demo

### LINKED ISSUE #6288: Card: Events view demo

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #6287 
Closes #6288

### CODE REVIEW

**Review by mcarrano — COMMENTED** [review]
These look good @tlabaj . But in comparing with the core implementations, I see a difference in font sizes for the log view card. I also referenced the original design and it looks like your implementation is closer to that. @mattnolting can you take a look at this? Do we need to fix the core implementation? Here is the design I'm looking at for reference:

Events card

**Comment by mcoker** [comment]
@mcarrano to confirm, to match the design we should also update the font size in the view card in this PR, as well as add a divider between the card body/footer? It would look like this:

Screen Shot 2021-09-10 at 10 22 23 AM

**Comment by mcarrano** [comment]
@tlabaj @mcoker Yes, that's correct.

**Comment by mcoker** [comment]
Created a core issue to update the demos - https://github.com/patternfly/patternfly/issues/4368

File: packages/react-core/src/demos/CardDemos.md [hunk 1]
```diff
@@ -365,51 +359,312 @@ CardDetailsDemo = () => {
...
+                Successfully assigned default/example to ip-10-0-130-149.ec2.internal
+              </DescriptionListDescription>
+              <DescriptionListDescription>
+                <date className="pf-u-color-200 pf-u-font-size-sm">Jun 17, 11:13 am</date>
               </DescriptionListDescription>
             </DescriptionListGroup>
             <DescriptionListGroup>
-              <DescriptionListTerm>
-                OpenShift Version
+              <DescriptionListTerm className=" pf-u-font-size-lg">
```
Comment (mcoker) [inline packages/react-core/src/demos/CardDemos.md:629]:
```suggestion
              <DescriptionListTerm>
```

File: packages/react-core/src/demos/CardDemos.md [hunk 2]
```diff
@@ -365,51 +359,312 @@ CardDetailsDemo = () => {
...
               </DescriptionListTerm>
+              <DescriptionListDescription>Pulling image "openshift/hello-openshift"</DescriptionListDescription>
               <DescriptionListDescription>
-                4.5.0.ci-2020-06-16-015026
+                <date className="pf-u-color-200 pf-u-font-size-sm">Jun 17, 10:59 am</date>
               </DescriptionListDescription>
             </DescriptionListGroup>
             <DescriptionListGroup>
-              <DescriptionListTerm>Update Channel</DescriptionListTerm>
+              <DescriptionListTerm className=" pf-u-font-size-lg">
```
Comment (mcoker) [inline packages/react-core/src/demos/CardDemos.md:645]:
```suggestion
              <DescriptionListTerm>
```

File: packages/react-core/src/demos/CardDemos.md [hunk 3]
```diff
@@ -365,51 +359,312 @@ CardDetailsDemo = () => {
...
+                connection refused
+              </DescriptionListDescription>
+              <DescriptionListDescription>
+                <date className="pf-u-color-200 pf-u-font-size-sm">Jun 17, 11:02 am</date>
+              </DescriptionListDescription>
             </DescriptionListGroup>
             <DescriptionListGroup>
-              <DescriptionListTerm>
-                Provider
+              <DescriptionListTerm className=" pf-u-font-size-lg">
```
Comment (mcoker) [inline packages/react-core/src/demos/CardDemos.md:611]:
```suggestion
              <DescriptionListTerm>
```

File: packages/react-core/src/demos/CardDemos.md [hunk 4]
```diff
@@ -365,51 +359,312 @@ CardDetailsDemo = () => {
...
+          <CardTitle>
+            <Title headingLevel="h2" size="xl">
+              Events
+            </Title>
+          </CardTitle>
+        </CardHeader>
+        <CardBody>
+          <DescriptionList>
+            <DescriptionListGroup>
+              <DescriptionListTerm className=" pf-u-font-size-lg">
```
Comment (mcoker) [inline packages/react-core/src/demos/CardDemos.md:592]:
```suggestion
              <DescriptionListTerm>
```

**Review by mcoker — CHANGES_REQUESTED** [review]

**Review by mcoker — APPROVED** [review]
LGTM!

**Review by mcarrano — APPROVED** [review]
These look great!

**Review by kmcfaul — APPROVED** [review]
lgtm

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**